File: Mario Project/marioForCOLE.cpp
#include <iostream>
#include <math.h>
#include <string.h>
#include <time.h>
#include <stdlib.h>
#include <stdio.h>
using namespace std;

int main()
{
    int height;
    printf("Give me a height: ");
    cin >> height;

  
    
    for(int width = height; width >= 1; width--)
    {

        int space = width - 1;
        for(int spaces = width - 1; spaces >= 1; spaces--)
        {
            printf(" ");
        }
        for(int hash = height - space; hash >= 1; hash--)
        {
            printf("#");
        }
        printf("\n");
        
    }
    
}

File: Mario Project/marioforcole.py
import sys

height = -1
while(height > 23 or height < 0):
    height = int(input("Give me height: "))

i = height
while(i >= 1):

    for spaces in range(0, i - 1):
        print(" ", end='')

    for hashes in range(i - 1, height):
        print("B", end='')
    print("\n", end='')
    i-=1
